config/prod/nf-osi-processing.yaml

```diff
@@ -0,0 +1,28 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v5.yaml
+stack_name: nf-osi-processing
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "NF OSI"
+  # The resource owner
+  OwnerEmail: "sara.gosline@sagebase.org"
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.2xlarge"
+  # (Optional) ID of the base AMI (supported distros: AWS linux)
+  AMIId: "ami-0d051218cbaf3405a"      # AMI with my R packages
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "100"
+  # (Optional) Run EC2 in a subnet (valid values: 'PrivateSubnet' or 'PublicSubnet')
+  VpcSubnet: "PublicSubnet"
+  # (Optional) Expose a port to incoming traffic (valid range: 1-65535)
+  OpenPort: "80"
```

Comment:
Is there a reason why you want to make this a public ec2 and open a port?  If so could you please add that info to the commit message?

Reply:
No.